### PR TITLE
fix!: import.meta not supported, drop support of Node<14

### PR DESCRIPTION
**Pull request by hannoeru**
#4 Change esbuild target to es2020 and add a notation on README.

### CODE REVIEW

File: src/build.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export async function build({ script = 'sync', mock = false } = {}) {
 
   const ssrConfig: UserConfig = {
     esbuild: {
-      target: 'es2018',
+      target: 'es2020',
     },
```
Comment (antfu) [inline src/build.ts:43]: We can just remove this section to align with client build (up to user's config, if any)

**Comment by antfu** [comment]
Great, thanks!

**Comment by tohagan** [comment]
Forcing Node >= 14 will limit the cloud hosting services that can support Vite. 
I was considering using Vite for a project but this will kill it.

**Comment by antfu** [comment]
Don't know there is anything we can do here. If you found a way to make it work for v12 or v10, would you mind drafting a PR for it? Thanks.

**Comment by sibbng** [comment]
I had an issue with that engine limitation. I don't know is that fix still applies to latest vite version or not.

While deploying an app to Netlify build has failed because of that.

I bypassed engine check setting that env variable on Netlify console and had no issue while deploying. But my app was just a single page app and doesn't even import any deps.
image

**Pull request merged**